apps/topicmaps/bplan-auskunft/src/app/components/BPlanInfo.tsx

```diff
@@ -12,7 +12,7 @@ import L from 'leaflet';
 //import type {TopicMapContext} from 'react-cismap';
 import { TopicMapContext } from 'react-cismap/contexts/TopicMapContextProvider';
 import { UIDispatchContext } from 'react-cismap/contexts/UIContextProvider';
-import { InfoBoxText } from '@carma-collab/wuppertal/bplan-auskunft';
+//import { InfoBoxText } from '@carma-collab/wuppertal/bplan-auskunft'; // TODO FIX IMPORT
```

Comment:
@d4v3000 are there some collab/commons for this?
cant find the imports. needs to be added back in ASAP